client/telemetry/src/lib.rs

```diff
@@ -101,6 +101,54 @@ pub struct ConnectionMessage {
 	pub startup_time: String,
 	/// Node's network ID.
 	pub network_id: String,
+
+	/// Node's OS.
+	pub target_os: String,
+
+	/// Node's ISA.
+	pub target_arch: String,
+
+	/// Node's target platform ABI or libc.
+	pub target_env: String,
+
+	/// Node's software and hardware information.
+	pub sysinfo: Option<SysInfo>,
+
+	/// Node's hardware benchmark results.
+	pub hwbench: Option<HwBench>,
```

Comment:
The first 3 could stay, but the rest should be removed from the default connection message. This could just be optional messages if a chain is using the benchmarking extension?